### PR TITLE
fix two snowball crashes

### PR DESCRIPTION
**Pull request by DS-Minetest**

### CODE REVIEW

**Comment by DS-Minetest** [comment]
To be merged?

File: src/snowball.lua [hunk 1]
```diff
@@ -174,7 +174,7 @@ function snow_snowball_ENTITY.on_step(self, dtime)
 	end
 	for _,v in pairs(minetest.get_objects_inside_radius(pos, 1.73)) do
 		if v ~= self.object then
-			local entity_name = v:get_luaentity().name
+			local entity_name = (v:get_luaentity() or {}).name
```
Comment (HybridDog) [inline src/snowball.lua]: Wann passiert der Absturz?

**Comment by Splizard** [comment]
Does this still fix any crashes? I have not noticed any crashes by throwing snowballs around.

**Comment by HybridDog** [comment]
The problem was already fixed in https://github.com/Splizard/minetest-mod-snow/commit/fd111402d53e4e1849dfbbcab91a05e0c1293888#diff-c8fdbcb744b02bb74d8d4df5e6c034e5

**Comment by HybridDog** [comment]
I've added the duplicate label although DS-Minetest may have provided the fix first.

**Pull request closed without merging**